solution/ui/regulations/eregs-vite/src/components/subjects/PolicySelections.vue

```diff
@@ -29,48 +35,20 @@ const removeClick = () => {
         },
     });
 };
-
-const selections = ref([]);
-
-watch(
-    () => selectedParams.paramString,
-    async () => {
-        selections.value = selectedParams.paramsArray
-            .filter((param) => FilterTypesDict[param.type])
-            .map((param) => ({
-                label: FilterTypesDict[param.type],
-                id: param.id,
-                type: param.type,
-                name: param.name,
-            }))
-            .sort((a, b) => {
-                // sort by label and then by name
-                if (a.label < b.label) return -1;
-                if (a.label > b.label) return 1;
-                if (a.name.toLowerCase() < b.name.toLowerCase()) return -1;
-                if (a.name.toLowerCase() > b.name.toLowerCase()) return 1;
-                return 0;
-            });
-    }
-);
```

Comment:
Originally, the Subjects page allows multiple subjects to be selected.  It was also a possibility that other filters (such as Category) would also be listed as chips alongside subjects at the top of the left sidebar.

However, this page evolved to allow only a single Subject to be selected at a time, and other filters have their own elements for selection and display.  (For example, the Categories dropdown at the top of the results pane allows selection and displays the selected category.)

Since only single subjects can be selected simultaneously, the sorting logic above is no longer necessary.  Additionally, we no longer have to watch for a change in the selectedParams object: we are now able to pass in the selected subject as a prop, and the component re-renders as that prop changes.